### ISSUE TITLE
Android: java.time package not work on SDK < 26

### ISSUE THREAD

**Issue by ltdangkhoa** [builttoroam/device_calendar#330]
This plugin use flutter_native_timezone which use java.time package for ZoneId. That will lead to the crash on device with lower api (<26)

```
E/AndroidRuntime( 5106): java.lang.NoClassDefFoundError: Failed resolution of: Ljava/time/ZoneId;
E/AndroidRuntime( 5106): 	at com.whelksoft.flutter_native_timezone.FlutterNativeTimezonePlugin.onMethodCall(FlutterNativeTimezonePlugin.kt:37)
E/AndroidRuntime( 5106): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:233)
E/AndroidRuntime( 5106): 	at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:84)
E/AndroidRuntime( 5106): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:869)
E/AndroidRuntime( 5106): 	at android.os.MessageQueue.nativePollOnce(Native Method)
```

Will there be any solution for this issue or we need to setup the minSdkVersion to 26?

**Comment by thomassth** [comment]
We probably should upgrade our min SDK.

Is there any specific cases that needs a lower SDK version to run?

**Comment by thomassth** [comment]
> I think making device_calender depends on flutter_native_timezone `2.0.0` will resolve this issue. [pinkfish/flutter_native_timezone#25](https://github.com/pinkfish/flutter_native_timezone/issues/25)

Then it's fixed with #345. Let me know if it isn't.
